docs/getting-started/installation/intellij.md

```diff
@@ -0,0 +1,6 @@
+# IntelliJ IDEA
+Rather than VS Code, when developing with kotlin the reccomended IDE is IntelliJ. IntelliJ offers much better autocomplete and syntax highlighting when using Kotlin and Java together (which we always are, as WPILib is Java)
```

Comment:
recommended

Kotlin should _probably_ be capitalized everywhere?

> (which we always are, as WPILib is Java)

I think this is a weird way of saying 'the JVM portion of WPILib is written in Java'. You could probably leave out this tidbit.